test/MUXControlsTestApp/InfoBarTestPage.xaml.cs

```diff
@@ -0,0 +1,214 @@
+        String closeButtonContent;
+        bool open;
+
+
+        public InfoBarTestPage()
+        {
+            this.InitializeComponent();
+        }
+
+        private async void Test_ActionButtonClick(object sender, RoutedEventArgs e)
+        {
+            await new MessageDialog("Thank you, mate").ShowAsync();
+        }
+
+        private void ComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            string severityName = e.AddedItems[0].ToString();
+
+            switch (severityName)
+            {
+                case "Critical":
+                    severity = InfoBarSeverity.Critical;
+                    break;
+                case "Warning":
+                    severity = InfoBarSeverity.Warning;
+                    break;
+                case "Informational":
+                    severity = InfoBarSeverity.Informational;
+                    break;
+                case "Success":
+                    severity = InfoBarSeverity.Success;
+                    break;
+                case "Default":
+                    severity = InfoBarSeverity.Default;
+                    break;
+                case "None":
+                    severity = InfoBarSeverity.None;
+                    break;
+            }
+        }
+
+        private void SeverityButton_Click(object sender, RoutedEventArgs e)
+        {
+            Test.Severity = severity;
+        }
+
+        private void IconComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            string iconName = e.AddedItems[0].ToString();
+
+            switch (iconName)
+            {
+                case "Pin Icon":
+
+                    SymbolIconSource sym2 = new SymbolIconSource();
+                    sym2.Symbol = Symbol.Pin;
+
+                    icon = (IconSource)sym2;
+                    break;
+                case "No Icon":
+                    SymbolIconSource sym3 = new SymbolIconSource();
+                    sym3.Symbol = new Symbol();
+                    icon = sym3;
+                    break;
+
+            }
+        }
+
+        private void IconButton_Click(object sender, RoutedEventArgs e)
+        {
+            Test.IconSource = icon;
+        }
+
+        private void TitleComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            string iconName = e.AddedItems[0].ToString();
+
+            switch (iconName)
+            {
+                case "Short Title":
+                    title = "Short Title.";
+                    break;
+                case "Long Title":
+                    title = "Long Title. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.";
+                    break;
+                case "No Title":
+                    title = "";
```

Comment:
also test title=null

Reply:
Good idea. Changed title, actionbuttoncontent, closebuttoncontent, and message to null instead of an empty string.